src/FastRoute/src/FastRouteKernel.cpp

```diff
@@ -361,18 +370,24 @@ void FastRouteKernel::runAntennaAvoidanceFlow()
     restorePreviousCapacities(_minRoutingLayer);
 
     _fastRoute->initAuxVar();
-    _fastRoute->run(newRoute);
-    addRemainingGuides(&newRoute);
+    _fastRoute->run(*newRoute);
```

Comment:
after you merge this I want to reorganize antenna fixing as a separate command instead of an arg to fastroute. the fastroute command is too overloaded